backend/src/core/utils/email-address.ts

```diff
@@ -0,0 +1,24 @@
+import psl from 'psl'
+
+/**
+ * Extract the domain of an email address
```

Comment:
Maybe it's more readable to explicitly mention that the extracted domain excludes the subdomain?
(If I didn't understand the code wrongly)
```suggestion
 * Extract the domain of an email address, excluding its subdomain
 * e.g. person@sub.agency.gov.sg -> agency.gov.sg
```